### PR TITLE
Embed the KOTS CLI binary and use it to install the app

### PR DESCRIPTION
**Pull request by sgalsaleh**
Embeds the KOTS CLI binary and uses it to install the app in both online and airgap mode.

[SC-101172](https://app.shortcut.com/replicated/story/101172)

### CODE REVIEW

File: pkg/airgap/version_test.go [hunk 1]
```diff
@@ -40,3 +44,82 @@ func TestAirgapBundleVersions(t *testing.T) {
 		})
 	}
 }
+
+func createTarballFromDir(rootPath string, additionalFiles map[string][]byte) io.Reader {
```
Comment (sgalsaleh) [inline pkg/airgap/version_test.go:48]: this was moved from the deleted `pkg/airgap/app_configmaps_test.go` file.

File: Makefile [hunk 2]
```diff
@@ -2,9 +2,10 @@ VERSION ?= $(shell git describe --tags --dirty)
 UNAME := $(shell uname)
 ARCH := $(shell uname -m)
 APP_NAME = embedded-cluster
+KOTS_VERSION = v1.108.2
 ADMIN_CONSOLE_CHART_URL = oci://registry.replicated.com/library
 ADMIN_CONSOLE_CHART_NAME = admin-console
-ADMIN_CONSOLE_CHART_VERSION = 1.108.1
+ADMIN_CONSOLE_CHART_VERSION = $(subst v,,$(KOTS_VERSION))
```
Comment (laverya) [inline Makefile:8]: you'll need to update the dependency update github action to handle this and not undo your work

Comment (sgalsaleh) [reply Makefile:8]: good catch.

Comment (sgalsaleh) [reply Makefile]: fixed.

**Review by laverya — APPROVED** [review]

**Review by cbodonnell — APPROVED** [review]

**Pull request merged**